EFTYPE is a glibc specific error type.

This patch fixed build error in libcouchbase.

See-Also: https://github.com/libuv/libuv/pull/1836

--- a/include/uv.h
+++ b/include/uv.h
@@ -142,7 +142,6 @@
   XX(EHOSTDOWN, "host is down")                                               \
   XX(EREMOTEIO, "remote I/O error")                                           \
   XX(ENOTTY, "inappropriate ioctl for device")                                \
-  XX(EFTYPE, "inappropriate file type or format")                             \
 
 #define UV_HANDLE_TYPE_MAP(XX)                                                \
   XX(ASYNC, async)                                                            \
